rpcs3/Emu/RSX/Overlays/HomeMenu/overlay_home_menu.cpp

```diff
@@ -171,31 +172,13 @@ namespace rsx
 			this->on_close = std::move(on_close);
 			visible = true;
 
-			auto& list_thread = g_fxo->get<named_thread<home_menu_dialog_thread>>();
```

Comment:
home_menu_dialog_thread can now be removed

Reply:
Done.